Patches/Alpha Animals/DamageDef/AlphaAnimals_CE_Patch_DamageDef.xml

```diff
@@ -0,0 +1,95 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<Patch>
+
+	<Operation Class="PatchOperationFindMod">
+		<mods>
+		<li>Alpha Animals</li>
+		</mods>
+
+			<match Class="PatchOperationSequence">
+				<success>Always</success>

```

Comment:
I believe it would require a followup patch for every iteration of the patch where a new creature or bodyDef is added, since the creator usually adds monsters every month or so (otherwise it's gonna run okay but with a vanilla behaviour)/

It's updated to the latest version as of 25/04. The latest change afaik didn't change much in term of XML's so I believe it should be okay.

Reply:
Did you mean to reply to https://github.com/CombatExtended-Continued/CombatExtended/pull/65#pullrequestreview-404514327? You can do that by adding a comment directly on the PR ;)

Reply:
Thanks. I'll do this in a short while.